bench/tx-generator/configuration-default.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
`bench/tx-generator` is basically the root directory of the cabal package.
Lets just move that file to `bench/` or somewhere else.